app/models/rules_factory.rb

```diff
@@ -4,10 +4,18 @@ def initialize(auction)
   end
 
   def create
+    eligibility = if auction.small_business?
```

Comment:
Oh I don't know. It just feels a bit weird since it's a relatively abstract noun and I don't know if we should be considering what the class is for (although I can only think of something ugly like `VendorGroup` and in procurement these are known as `SetAside`) or something else. Never mind me